src/model/series/custom-pane-view.ts

```diff
@@ -78,6 +78,15 @@ export class SeriesCustomPaneView extends SeriesPaneViewBase<
 		);
 	}
 
+	public getConflationReducer(): ((items: readonly unknown[]) => unknown) | undefined {
+		const reducer = this._paneView.conflationReducer; // eslint-disable-line @typescript-eslint/unbound-method
+		if (!reducer) {
+			return undefined;
+		}
+
+		return reducer as (items: readonly unknown[]) => unknown;
+	}
```

Comment:
```suggestion
	public get conflationReducer(): ((items: readonly CustomConflationContext<unknown, CustomData<unknown>>[]) => CustomData<unknown>) | undefined {
		// eslint-disable-next-line @typescript-eslint/unbound-method
		return this._paneView.conflationReducer;
	}
```
Using a getter simplifies the code using this, and also simplifies this code and it's types.